test/schema_test.rb

```diff
@@ -7,7 +7,8 @@ class SchemaTest < MiniTest::Unit::TestCase
   def test_to_s
     schema = Heroics::Schema.new(SAMPLE_SCHEMA)
     assert_equal(
-      '#<Heroics::Schema description="Sample schema for use in tests.">',
+      '#<Heroics::Schema description="Sample schema for use in tests
```

Comment:
Maybe this should use `\n` like the other examples/tests? That way it is more explicit/clear (and maybe less likely to inadvertently be broken)